src/main/groovy/ai/digital/integration/server/tasks/worker/SetWorkersLogbackLevelsTask.groovy

```diff
@@ -0,0 +1,56 @@
+package ai.digital.integration.server.tasks.worker
+
+import ai.digital.integration.server.domain.Worker
+import ai.digital.integration.server.util.DbUtil
+import ai.digital.integration.server.util.FileUtil
+import ai.digital.integration.server.util.LogbackConfigs
+import ai.digital.integration.server.util.WorkerUtil
+import groovy.xml.QName
+import groovy.xml.XmlUtil
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+import static ai.digital.integration.server.constant.PluginConstant.PLUGIN_GROUP
+
+class SetWorkersLogbackLevelsTask extends DefaultTask {
```

Comment:
Code is almost identical.
Let's reuse the code

Reply:
added LogbackUtil for common parts